content/en/users/training/infrastructure/_index.md

```diff
@@ -1,20 +1,166 @@
 ---
 title: "Training Infrastructure"
-linkTitle: "Training infrastructure"
 type: docs
-weight: 10
+weight: 100
 description: >
-  Training infrastructure
+  The training infrastructure on EGI Cloud 
 ---
 
-{{% pageinfo %}}
-This is a placeholder page.
-{{% /pageinfo %}}
+The training infrastructure is a resource pool within the EGI Federated
+Cloud infrascture providing IaaS as well ass access services (login,
```

Comment:
```suggestion
Cloud infrascture providing IaaS as well as access services (login,
```

Reply:
Duh, missed your comment, we will keep an ass in the doc for some more time ;)

Reply:
Once the code owner PR will be merged I will make another one to fix this and see if codeowners conf is working.

Reply:
Done in #45